migrations/2023-01-11-134448_add_col_to_merchant_account/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Oh, by the way, please update the directory name to be `add_metadata_to_merchant_account` instead of `add_col_to_merchant_account`.